src/components/editor/Bindings/FieldSelection/index.tsx

```diff
@@ -89,7 +90,15 @@ const mapConstraintsToProjections = (
                     constraint.type
                 );
 
-                selectionType = includeRequired ? 'include' : null;
+                const excludeRequired = evaluateRequiredExcludedFields(
+                    constraint.type
+                );
```

Comment:
Can you make this function call down below in the ternary? That way we can skip if not needed.

Reply:
No, because it is used to set the `disabled` property of the _Include_ button as well as within the click handler for the _Exclude_ button.